tests/MongoDB.Driver.Tests/Specifications/retryable-reads/RetryableReadsTestRunner.cs

```diff
@@ -186,11 +187,13 @@ private void InsertGridFsData(BsonDocument shared)
 
         private DisposableMongoClient CreateDisposableClient(BsonDocument test, EventCapturer eventCapturer)
         {
-            return DriverTestConfiguration.CreateDisposableClient((MongoClientSettings settings) =>
-            {
-                ConfigureClientSettings(settings, test);
-                settings.ClusterConfigurator = c => c.Subscribe(eventCapturer);
-            });
+            return DriverTestConfiguration.CreateDisposableClient(
+                (MongoClientSettings settings) =>
+                {
+                    settings.HeartbeatInterval = TimeSpan.FromMilliseconds(5); // may be overwritten in the next step in future
```

Comment:
I will create a spec ticket to describe this change

Reply:
https://github.com/mongodb/specifications/pull/820 I didn't touch this particular change (I also added this change on the lowest level in tests), since the mentioned PR is not applied yet. So, not 100% sure what level this change should be applied on.